### DISCUSSION TITLE
Stuck at setting up infinitely

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [Slackadays/Clipboard#205] [Q&A] [answered]
I've been using `cb` for a long time. Unfortunately, somehow someday, it's just broken: it starts setting up infinitely, no matter which shell I use and what command I give.

```log
$ cb <whatever-the-command>
⬤ Setting up... %?, 128s elapsed █████████████████████████▒▒▒▒▒▒▒▒▒▒▒▒▒▒
```

Both rebooting and reinstalling cannot fix it.

**Comment by YDX-2147483647** [comment, marked as answer]
(Today, I can't bear it anymore.)

Search the temp directory, and locate the file `$env:TEMP/Clipboard/0/metadata/lock`. Delete it.

**Comment by Slackadays** [comment]
This behavior has been improved in https://github.com/Slackadays/Clipboard/commit/ef88c1704cceaa8e173fcddf0d4ec8329d743610

**Reply by YDX-2147483647** [reply]
Thank you. Looking forward to the next release!
